app/routes/definitions.py

```diff
@@ -356,6 +357,54 @@ async def create_credential_definition(
                     rev_reg_id=revoc_reg_creation_result.revoc_reg_id, state="active"
                 )
                 credential_definition_id = active_rev_reg.result.cred_def_id
+                try:
+                    async with client_from_auth(auth) as aries_controller:
+                        bound_logger.debug(
+                            "Publishing rev reg entry for: rev_reg_id: {} and conn_id: {}",
+                            revoc_reg_creation_result.revoc_reg_id,
+                            endorser_connection_id,
+                        )
+
+                        await aries_controller.revocation.publish_rev_reg_entry(
+                            rev_reg_id=revoc_reg_creation_result.revoc_reg_id,
+                            conn_id=endorser_connection_id,
+                            create_transaction_for_endorser=True,
```

Comment:
Since only an issuer can create a `cred_def `  it  _should_  have a connection with the endorser (since issuers on the stack has a connection with endorser). 
If it does not have a connection with the endorser it wont be able to publish the entry to the ledger since the issuer's write to the ledger needs to be endorsed. 
So maybe if `has_connection = false ` we should create a connection for them.

Reply:
Dis waar - ek dink net aan die geval waar iets verander, soos die endorser skuif, en dan lyk dit nie meer asof daar 'n connection is nie. In daai geval moet ons seker error, want issuers moet 'n connection he, en net issuers kan hierdie route call. Ek dink ons later 'n "create endorser connection" function implement en dit dynamically doen, of een of ander manier implement. Maar vir nou, ons moet of error as daar nie 'n connection is nie, of ons moet net se `create_transaction_for_endorser=has_connections`. Ek dink error is seker beter, dan vang is ons in toekoms as endorser connections verwyder word